client/src/extension.ts

```diff
@@ -164,6 +166,7 @@ function startClient(context: ExtensionContext, elmWorkspace: Uri) {
 
   // Start the client. This will also launch the server
   languageClient.start();
+  Package.activatePackage()
```

Comment:
We probably need to pass the directory that the `elm.json` is in here, so that we can determine on which one to run the commands?
Not sure if commands will correctly report, that they belong to this instance? On the server side we needed to make the commandnames unique over all instances.

Reply:
> Not sure if commands will correctly report, that they belong to this instance?

Yeah, that was my mistake, haven't correctly understood what's going on during activation. Fixed that.

> We probably need to pass the directory that the elm.json is in here, so that we can determine on which one to run the commands?

This is interesting question... but we don't need to do anything. Because we use terminal integration to run `elm install <package>` we depend on the default behaviour of it in VSCode - it's spawning terminal in root of currently active folder in the workspace. 
So if you have 2 workspace folders `elmA` and `elmB` - if you're currently in any file from `elmA` terminal will start for it (and hence we'll install package for this project), and if you're in any file from `elmB` terminal will start there.

Reply:
I just tried that with the lastest version you pushed here. It still ends up spawning in the topmost directory and thus not being able to find any `elm.json`.

![Screenshot from 2019-08-06 20-06-37](https://user-images.githubusercontent.com/5943908/62564808-cf531300-b885-11e9-9bc9-abe7eea26a0f.png)
)

Reply:
I've meant VSCode multi-folder workspaces (https://code.visualstudio.com/docs/editor/multi-root-workspaces) rather than just single-folder workspace containing multiple Elm projects.

Hmm... yes, that's tricky. Is that (having multiple Elm "projects" in single workspace/repository) something that happens in practice? I'm not doing a lot of Elm so I'm not sure what are community practices around things like that

Reply:
As far as I can tell, there are some projects out there, that do it like that. It would also help with some tuturials, where each step would be a complete example in a subfolder.

Anyway, I think it would be fine to merge this and track this behavior in an issue for now, what do you think?

Reply:
Yeah, I think we may merge this, and figure out the problematic behaviour later